lib/src/default_extension_map.dart

```diff
@@ -419,6 +419,7 @@ const Map<String, String> defaultExtensionMap = <String, String>{
   'm3u8': 'application/vnd.apple.mpegurl',
   // Source: https://www.rfc-editor.org/rfc/rfc4337#page-3
   'm4a': 'audio/mp4',
+  'm4b': 'audio/mp4',
```

Comment:
like above, please add a comment referencing the corresponding RFC, etc

Reply:
Both .m4a and .m4b files are MP4 files which include audio only. The only difference is metadata - m4b supports bookmarks, which are needed for audiobooks. 

So the same RFC applies to both file extensions. I updated the link as the old one doesn't lead anywhere.